docs/tutorials/feature-matrix.rst

```diff
@@ -3,34 +3,36 @@ Feature Matrix
-|Fancy Speed Effects   |  ✖   |   ✖   |   ✖    |   ✖    |   ✖   |   ✖    |   ✖   |    ✖     |
-+----------------------+------+-------+--------+--------+-------+--------+-------+----------+
-|Color Decision List   |  ✔   |   ✔   |   ✖    |   ✖    |   ✖   |   ✖    |  N/A  |    ✖     |
-+----------------------+------+-------+--------+--------+-------+--------+-------+----------+
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Feature                  | OTIO | EDL   |FCP7 XML| FCP X  | AAF   |   RV   | ALE   |GStreamer |
++=========================+======+=======+========+========+=======+========+=======+==========+
+|Single Track of Clips    |  ✔   |   ✔   |   ✔    |   ✔    |   ✔   |  W-O   |   ✔   |    ✔     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Multiple Video Tracks    |  ✔   |   ✖   |   ✔    |   ✔    |   ✔   |  W-O   |   ✔   |    ✔     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Audio Tracks & Clips     |  ✔   |   ✔   |   ✔    |   ✔    |   ✔   |  W-O   |   ✔   |    ✔     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Gap/Filler               |  ✔   |   ✔   |   ✔    |   ✔    |   ✔   |   ✔    |   ✖   |    ✔     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Markers                  |  ✔   |   ✔   |   ✔    |   ✔    |   ✖   |  N/A   |   ✖   |    ✔     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Nesting                  |  ✔   |   ✖   |   ✔    |   ✔    |   ✔   |  W-O   |   ✔   |    ✔     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Transitions              |  ✔   |   ✔   |   ✖    |   ✖    |   ✔   |  W-O   |   ✖   |    ✔     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Audio/Video Effects      |  ✖   |   ✖   |   ✖    |   ✖    |   ✖   |  N/A   |   ✖   |    ✔     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Linear Speed Effects     |  ✔   |   ✔   |   ✖    |   ✖    |   R-O |   ✖    |   ✖   |    ✖     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Fancy Speed Effects      |  ✖   |   ✖   |   ✖    |   ✖    |   ✖   |   ✖    |   ✖   |    ✖     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Color Decision List      |  ✔   |   ✔   |   ✖    |   ✖    |   ✖   |   ✖    |  N/A  |    ✖     |
++-------------------------+------+-------+--------+--------+-------+--------+-------+----------+
+|Image Sequence Reference |  ✔   |   ✖   |   ✖    |   ✖    |   ✖   |   ✔    |   ✖   |    ✖     |
```

Comment:
```suggestion
|Image Sequence Reference |  ✔   |   ✖   |   ✖    |   ✖    |   ✖   |   W-O    |   ✖   |    ✖     |
```

(because RV technically is write only)

Reply:
Ah, god catch! I'll update.